app/scheduling/directives/termCalendar.js

```diff
@@ -104,34 +104,104 @@ schedulingApp.directive("termCalendar", this.termCalendar = function ($rootScope
 				});
 			};
 
+			// supply a color and amount to shift it by
```

Comment:
Add a comment as to the units we're shifting by. Are these percents? Or are they out of 255?